core/base/assignmentSolver/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+ttk_add_base_library(assignmentSolver
+  SOURCES 
+    AssignmentSolver.cpp
+  HEADERS 
+    AssignmentSolver.h
+    AssignmentAuction.h
+    #AssignmentExhaustive.h
```

Comment:
Why not building the exhaustive version?

Reply:
Indeed, it is a mistake, will be fixed in the next commit!